rclcpp/test/rclcpp/test_publisher_with_type_adapter.cpp

```diff
@@ -179,20 +145,22 @@ TEST_F(TestPublisher, conversion_exception_is_passed_up) {
         (void)msg;
       };
 
-    initialize(options);
+    auto node = std::make_shared<rclcpp::Node>("my_node", "/ns", options);
     auto pub = node->create_publisher<BadStringTypeAdapter>("topic_name", 1);
     // A subscription is created to ensure the existence of a buffer in the intra proccess
     // manager which will trigger the faulty conversion.
     auto sub = node->create_subscription<rclcpp::msg::String>("topic_name", 1, callback);
     EXPECT_THROW(pub->publish(1), std::runtime_error);
+
+    node.reset();
```

Comment:
same comment about this line being unnecessary

Reply:
Fixed in the latest (I ended up squash-merging this into the appropriate commit).